src/sidebar/config/host-config.js

```diff
@@ -78,35 +78,47 @@ export function hostPageConfig(window) {
   // It is assumed we should expand this list and coerce and eventually
   // even validate all such config values.
   // See https://github.com/hypothesis/client/issues/1968
+
+  /** @type {Record<string, (value: unknown) => unknown>} */
```

Comment:
This type is "conservative", as the results from the various methods in this object are not `unknown`, but it is good enough for the code below.